### PR TITLE
chore(deps): upgrade sync-monorepo-packages to v1.0.0

### PR DESCRIPTION
**Pull request by boneskull**
This also means we can remove `packages` field from `lerna.json`

### CODE REVIEW

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* master
  * **PR #17811** <a href="https://app.graphite.dev/github/pr/appium/appium/17811" target="_blank">Graphite</a> 
    * **PR #17846** <a href="https://app.graphite.dev/github/pr/appium/appium/17846" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/appium/appium/17846?utm_source=stack-comment).

**Comment by boneskull** [comment]
(it targets #17811 due to the node version requirement of `sync-monorepo-packages`)

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "dev:types": "tsc -b --watch",
     "doctor": "node ./packages/doctor",
     "generate-docs": "lerna run --scope=appium generate-docs",
-    "prepare": "npm run build:types || true",
+    "prepare": "npm run build:types || true",
```
Comment (jlipps) [inline package.json:44]: does this add a space?

**Review by jlipps — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (boneskull) [reply package.json:44]: apparently

**Pull request merged**